Description: add missing examples/ dir in Debian package
 As described in this bug report
 <URL: https://github.com/samaaron/sonic-pi/issues/1229>,
 the Debian package for Sonic Pi 2.10.0 was missing the
 examples/ directory during installation. 
Forwarded: not-needed
Author: Hanno Zulla <kontakt@hanno.de>
Last-Update: 2016-05-17

--- a/app/gui/qt/mainwindow.cpp
+++ b/app/gui/qt/mainwindow.cpp
@@ -1063,7 +1063,7 @@
     return QCoreApplication::applicationDirPath() + "/../../../../..";
 #else
     // On linux, CMake builds app into the build folder
-    return QCoreApplication::applicationDirPath() + "/../../../..";
+    return "/usr/lib/sonic-pi";
 #endif
 }
 
